presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -658,6 +667,10 @@ std::vector<std::string> PrestoServer::registerConnectors(
           fileName.substr(0, fileName.size() - kPropertiesExtension.size());
 
       auto connectorConf = util::readConfig(entry.path());
+      PRESTO_STARTUP_LOG(INFO)
+          << "Registered " << entry.path() << " properties:\n"
```

Comment:
nit: Registered connector properties from <path>:

Reply:
I have kept it this way to be consistent with the rest of the startup logs at 
https://github.com/prestodb/presto/blob/88968840b4449551871fe2953e1a37087cb5766c/presto-native-execution/presto_cpp/main/common/Configs.cpp#L210